tests/wpt/web-platform-tests/conformance-checkers/html/elements/title/empty-novalid.html

```diff
@@ -1,4 +1,3 @@
 <!doctype html>
 <!-- document with empty title element -->
 <meta charset=utf-8>
-<title></title>
```

Comment:
This test is explicitly testing empty title elements. Looks like we need to do something smarter.